lib/job-simulation.rb

```diff
@@ -1,19 +1,47 @@
-require './Stack.rb'
-require './Queue.rb'
+require_relative './Stack.rb'
+require_relative './Queue.rb'
 
 class JobSimulation
   attr_reader :workers, :waiting, :roll
 
   def initialize (jobs_available, job_seekers)
+    @workers = Stack.new
+    @waiting = Queue.new
+    job_seekers.times do |num|
+      @waiting.enqueue("wokrer ##{num + 1}")
+    end
+
+    jobs_available.times do
+    @workers.push(@waiting.front)
+    @waiting.dequeue
+    end
```

Comment:
Since `dequeue` returns the element that's removed, you could simplify above as follows:
```
jobs_available.times do
  @workers.push( @waiting.dequeue )
end
```